src/Activator.php

```diff
@@ -4,6 +4,12 @@
 class Activator {
 	public function __construct( $file ) {
 		register_activation_hook( $file, [ $this, 'activate' ] );
+
+		add_filter( 'plugin_action_links_' . plugin_basename(SLIM_SEO_FILE), function($links) {
+			$links[] = '<a href="' . admin_url( 'options-general.php?page=' . SLIM_SEO_SLUG ) . '">' . __('Settings') . '</a>';
+
+			return $links;
+		} );
```

Comment:
Can you please:

- Separate this function into a method to avoid using anonymous callback?
- Correct the spaces according to WP coding standards?
- Add textdomain `slim-seo` to the text "Settings"?
- Add `esc_url` to the `admin_url`?

Reply:
Thank you for the detailed hints. I still need to learn a lot when it comes to programming, and this really helped me. Hope it fits now.